Issue #49 patch: Template selection dialog keeps shifting on selection -...

ravirnjn88 opened the pull request:
Fixed the shifting of template selection dialogue on selection.
Screen shots: https://www.dropbox.com/s/7vrn6c7ap2x91et/BuildmLearn.jpg?dl=0

martinrotter commented:
OK, I will check today or tomorrow and eventually merge.

ravirnjn88 commented:
@martinrotter Please review the PR

martinrotter commented:
Will check today hopefully, I am totally busy.

2015-02-21 20:31 GMT+01:00 Ravi Ranjan <notifications@github.com>:

> @martinrotter <https://github.com/martinrotter> Please review the PR
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BuildmLearn/BuildmLearn-Toolkit/pull/50#issuecomment-75388271>
> .
>

martinrotter commented on src/gui/formnewproject.ui:
```diff
@@ -24,10 +27,28 @@
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
-      <string>Description of selected template</string>
+      <string>           Description of selected template        </string>
```
I really do not like those continuous spaces in this string. Fix this before I can even think about merging this pull request. It seems you try to hack centering the text of groupbox, which is not correct, each system does this on its own.

martinrotter commented on src/gui/formnewproject.ui:
```diff
@@ -6,13 +6,16 @@
    <string>Create new project</string>
   </property>
+  <property name="layoutDirection">
+   <enum>Qt::LeftToRight</enum>
```
Layout direction is always left to right, why you set it explicitly?

martinrotter commented on src/gui/formnewproject.ui:
```diff
@@ -24,10 +27,28 @@
+   <item row="0" column="0" rowspan="2">
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="title">
+      <string>          Available templates          </string>
```
Cannot accept those continuous spaces, sorry. :)

ravirnjn88 commented:
You are right, I am using those spaces for centering the text of the groupbox. While Using the alignment property (AlignCenter) it's not showing the result as expected.
Removed the spaces and corrected layout direction.

martinrotter commented:
Looks better. Merging to master.
